pkg/component/kubernetes/apiserverexposure/templates/envoyfilter-apiserver-proxy.yaml

```diff
@@ -141,7 +149,11 @@ spec:
       value:
         name: {{ .TargetClusterAPIServerProxy }}
         type: STATIC
-        connect_timeout: 1s
+        connect_timeout: 10s
```

Comment:
Why did we need to increase the `connect_timeout`? Asking because I can't infer this change from the PR description